packages/picker/src/Picker.ts

```diff
@@ -406,6 +412,15 @@ export class PickerBase extends SizedMixin(Focusable) {
                           ></sp-icon-alert>
                       `
                     : nothing}
+                ${this.loading
+                    ? html`
+                          <sp-progress-circle
+                              indeterminate
+                              aria-labelledby="label"
```

Comment:
`aria-hidden` below cancels this out, so we can exclude this line.

Reply:
`sp-progress-circle` throws a11y warnings if label is not provided, even when `aria-hidden` is present. Should we add a check for `aria-hidden` in this case too?